integration/kubernetes/pom.xml

```diff
@@ -148,6 +148,7 @@
                                     <path>${project.build.directory}/bundle</path>
                                 </paths>
                             </extraDirectories>
+                            <allowInsecureRegistries>true</allowInsecureRegistries>
```

Comment:
How is this change related?

Reply:
Nice catch, it is a leftover, I was seeing a message like below for our public images at Docker Hub and thought this might solve this so it shouldn't attempt to authenticate against that.

```
[INFO] The base image requires auth. Trying again for registry.hub.docker.com/hazelcast/hazelcast-jet:latest-snapshot...
```

Anyhow, the image builds fine without this property so I'll remove it.